.tools/rector/rector.php

```diff
@@ -23,6 +23,7 @@
     // this list will grow over time.
     // to make sure we can review every transformation and not introduce unseen bugs
     $parameters->set(Option::PATHS, [
+        'redaxo/src/addons/*/lib/',
```

Comment:
So greift es auch auf Dritt-Addons, die man lokal bei sich liegen hat.
Deswegen arbeiten wir meist mit expliziten Addonlisten in den configs.

Reply:
ggf. können wir die core addons mal irgendwo als konstante hinterlegen für diese cases?

Reply:
in den xml und neon dateien für psalm und phpstan würde uns das aber nicht helfen, oder?
Ich weiß gerade nicht, wie viele Stellen es gibt, wo man die liste importieren könnte.
So selten wie wir neue Addons hier erstellen ist das für mich nicht notwendig.
Ich habe aber auch nichts dagegen, wenn du es irgendwie auslagerst.